redis-test/test_strings.py

```diff
@@ -83,6 +85,20 @@ def test_echo():
     r = connect()
     assert r.echo('hello world') == 'hello world'
 
+def test_mset():
+    r = connect()
+    values = {}
+    for i in range(5):
+        key = random_string(10)
+        val = random_string(1000)
+        values[key] = val
+    assert r.mset(values) == True
```

Comment:
I'll commit this as-is, but a better test would not just write the keys - it would also read them back to see it actually works as expected and doesn't just ignore its arguments or do only the first one or anything like that.

Reply:
Fixed this in solving the conflicts.

```
    for k,v in values.items():
        assert r.get(k) == v
```